class AtaqueJaRealizado(Exception):
    def __init__(self):
        super().__init__("JÃ¡ houve uma batalha nessa rodada!")
